src/client/src/orangehrmRecruitmentPlugin/components/CandidateActionLayout.vue

```diff
@@ -110,7 +110,7 @@ export default {
       statuses: [
         {id: 1, label: this.$t('recruitment.application_initiated')},
         {id: 2, label: this.$t('recruitment.shortlisted')},
-        {id: 3, label: this.$t('recruitment.rejected')},
+        {id: 3, label: this.$t('leave.rejected')},
```

Comment:
oops. can change the group to general right?

Reply:
We have it in 5.0 leave